tinygrad/runtime/support/elf.py

```diff
@@ -17,11 +17,11 @@ def _strtab(blob: bytes, idx: int) -> str: return blob[idx:blob.find(b'\x00', id
   def _to_carray(sh, ctype): return (ctype * (sh.header.sh_size // sh.header.sh_entsize)).from_buffer_copy(sh.content)
   rel = [(sh, sh.name[4:], _to_carray(sh, libc.Elf64_Rel)) for sh in sections if sh.header.sh_type == libc.SHT_REL]
   rela = [(sh, sh.name[5:], _to_carray(sh, libc.Elf64_Rela)) for sh in sections if sh.header.sh_type == libc.SHT_RELA]
-  symtab = [_to_carray(sh, libc.Elf64_Sym) for sh in sections if sh.header.sh_type == libc.SHT_SYMTAB][0]
+  symtab, sl = [(_to_carray(sh, libc.Elf64_Sym), sh.header.sh_link) for sh in sections if sh.header.sh_type == libc.SHT_SYMTAB][0]
```

Comment:
this is because sometimes clang generates multiple strtabs (sh_link in symtab points to strtab that symtab is using)

Reply:
Is this tested?